.github/workflows/ci-build.yaml

```diff
@@ -341,7 +341,7 @@ jobs:
           # port 8080 which is not visible in netstat -tulpen, but still there
           # with a HTTP listener. We have API server listening on port 8088
           # instead.
-          make start-e2e-local &
+          make test-tools-image start-e2e-local &
```

Comment:
This should not be necessary - for local test, we don't need to build the image. If we want to build the image as part of the integration tests, we should parallelize the build in a dedicated step IMO.

Reply:
Did this change because I was trying to stop using `argoproj/argo-cd-ci-builder:v1.0.0` in https://github.com/argoproj/argo-cd/blob/master/test/fixture/testrepos/start-git.sh#L5-L5 .  Reverted it since some more work is required to stop using `argoproj/argo-cd-ci-builder:v1.0.0` image.